### PR TITLE
net-misc/trackma: version bump to 0.8.2

### PR DESCRIPTION
**Pull request by strikersh**

### CODE REVIEW

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: EAPI 7, please.

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+PYTHON_COMPAT=( python{3_5,3_6} )
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: Anything blocking py3.7?

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Open multi-site list manager for media tracking sites"
+HOMEPAGE="https://github.com/z411/trackma"
+SRC_URI="https://github.com/z411/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3"
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: Empty line above LICENSE. Also please verify license wrt https://devmanual.gentoo.org/general-concepts/licenses/index.html#gpl-x-vs-gpl-x+

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+ncurses cli gtk inotify qt5"
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: Sort flags by name.

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+REQUIRED_USE="|| ( cli gtk ncurses qt5 )"
+
+DEPEND="ncurses? ( dev-python/urwid[${PYTHON_USEDEP}] )
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: Also here sort by cat/pn, then by USE conditionals.

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+	)
+	qt5? (
+		dev-python/PyQt5[${PYTHON_USEDEP},gui,widgets,network]
+		dev-python/pillow[${PYTHON_USEDEP}]
+	)"
+RDEPEND="
+	!inotify? ( sys-process/lsof )
+	inotify? ( || ( dev-python/inotifyx dev-python/pyinotify ) )
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: Are you sure neither of them needs PYTHON_USEDEP?

**Comment by frstrikerman** [comment]
I will see for fixing those issue.
It's my first time making an ebuild so I mostly updated stuff.
I frequently use this package so I won't mind being a proxy maintainer for it

File: net-misc/trackma/trackma-0.8.2.ebuild (hunk 5 above)
Comment (frstrikerman) [reply net-misc/trackma/trackma-0.8.2.ebuild]: what do you mean by "pn"?
the soft has three different ui and so it need at least one of those

**Comment by mgorny** [comment]
Ok, cool. Please take your time and feel free to ping us if you need help.

**Comment by frstrikerman** [comment]
I'm not sure what to do about one of the change I posted a reply but it seams that you haven't seen it yet @mgorny

File: net-misc/trackma/trackma-0.8.2.ebuild (hunk 5 above)
Comment (mgorny) [reply net-misc/trackma/trackma-0.8.2.ebuild]: cat=category, pn=package name. In other words, the order should be: first unconditional dependencies, sorted by full package name, then all conditional dependencies, sorted by the flag name, and inside those groups again same rules.

```
dev-libs/bar
dev-libs/foo
sys-libs/baz
bar? ( app-misc/bar
    sys-misc/aaaa )
foo? ( app-misc/foo )
```

**Comment by mgorny** [comment]
I'm sorry, I have indeed missed that comment.

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="+cli gtk inotify ncurses qt5"
+REQUIRED_USE="|| ( cli gtk ncurses qt5 )"
+
+DEPEND="
+gtk? (
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: Please use one tab of indent for each level, i.e. add tab before all conditions.

File: net-misc/trackma/trackma-0.8.2.ebuild [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+	)
+ncurses? ( dev-python/urwid[${PYTHON_USEDEP}] )
+qt5? (
+		dev-python/PyQt5[${PYTHON_USEDEP},gui,widgets,network]
+		dev-python/pillow[${PYTHON_USEDEP}]
+	)"
+RDEPEND="
+	inotify? ( || ( dev-python/inotifyx[${PYTHON_USEDEP}] dev-python/pyinotify[${PYTHON_USEDEP}] ) )
```
Comment (mgorny) [inline net-misc/trackma/trackma-0.8.2.ebuild]: Now wrap the line, please. It's gotten too long.

**Comment by frstrikerman** [comment]
Is there anything else preventing it from being merged @mgorny ?

**Comment by juippis** [comment]
trackma is being removed from ::gentoo soon.

**Pull request closed without merging**